src/components/Report.tsx

```diff
@@ -87,9 +106,15 @@ export const Report = ({
   workflowMapping = defaultWorkflowMapping,
   children,
   className,
+  applicationInsightConnectionString,
+  SyncReportOpenedEventData,
+  issueArticleOpenEventData,
 }: {
   /** The report data should be compatible with the type definitions. */
   data: ReportData;
+  applicationInsightConnectionString?: string;
+  SyncReportOpenedEventData?: SyncReportOpenedEventDataType;
```

Comment:
why do we need to introduce those two new props?

Reply:
As we will be taking some information from users as not all our necessary  informations are not present in report, So we will be taking those props, As discussed over call will modify the code. Thanks.